### ISSUE TITLE
The critic observation (privileged infomation) is enough ?

### ISSUE THREAD

**Issue by mggeng** [Hellod035/LeggedLab#33]
Hello, thanks for your work. I want to konw wether the critic observation is enough. As shown in code, only footcontact and robot velocity is obtained by the critic network.
I don't konw if  more information  should be added for common bipedal humaniod locomotion, will the result be better? (only add obervation and no special tricks)

By the way, in isaaclab the obsterm is used. I am not sure if i can use obsertem binding to a function (like reward), and use obertem template.

Or Must i to write as this mannully?

```
        root_lin_vel = robot.data.root_lin_vel_b
        feet_contact = torch.max(torch.norm(net_contact_forces[:, :, self.feet_cfg.body_ids], dim=-1), dim=1)[0] > 0.5
        current_critic_obs = torch.cat(
            [current_actor_obs, root_lin_vel * self.obs_scales.lin_vel, feet_contact], dim=-1
        )
```

Best regards!
